src/lib/Component/EventSubscriber/RenderEventLegacyBridgeSubscriber.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace Ibexa\AdminUi\Component\EventSubscriber;
+
+use Ibexa\AdminUi\Component\Event\RenderGroupEvent;
+use Ibexa\AdminUi\Component\Event\RenderSingleEvent;
+use Ibexa\AdminUi\Component\Registry;
+use Ibexa\Contracts\TwigComponents\Event\RenderGroupEvent as TwigComponentsRenderGroupEvent;
+use Ibexa\Contracts\TwigComponents\Event\RenderSingleEvent as TwigComponentsRenderSingleEvent;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class RenderEventLegacyBridgeSubscriber implements EventSubscriberInterface
```

Comment:
`LegacyBridge` in the name gave mi shivers. xD Is there any reason why it's named like so? Besides:

```suggestion
final class RenderEventLegacyBridgeSubscriber implements EventSubscriberInterface
```